fast64_internal/oot/oot_level_parser.py

```diff
@@ -947,7 +947,7 @@ def parsePathList(
     pathData = getDataMatch(sceneData, pathListName, "Path", "path list")
     pathList = [value.replace("{", "").strip() for value in pathData.split("},") if value.strip() != ""]
     for pathEntry in pathList:
-        numPoints, pathName = [value.strip() for value in pathEntry.split(",")]
+        _, pathName = [value.strip() for value in pathEntry.split(",")]
```

Comment:
I think it's better to keep even unused things named, it serves as documentation / is more readable
```suggestion
        numPoints, pathName = [value.strip() for value in pathEntry.split(",")]
```